lib/api/web-element/commands/isVisible.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Determines if an element is visible.
+ *
+ * For more info on working with DOM elements in Nightwatch, refer to the
+ * <a href="https://nightwatchjs.org/guide/writing-tests/finding-interacting-with-dom-elements.html">Finding & interacting with DOM Elements</a> guide page.
+ *
+ * @example
+ * export default {
+ *   demoTest(browser: NightwatchAPI): void {
+ *     const result = browser.element('#main').isVisible();
+ *       .assert.equals(true);
+ *   },
+ *
+ *   async demoTestAsync(browser: NightwatchAPI): Promise<void> {
+ *     const result = await browser.element('#main').isVisible();
+ *     browser.assert.equal(result,true)
+ *   }
+ * }
```

Comment:
This example won't work (due to an obvious mistake). You should always try to run the examples yourself before putting them here.